website/docs/installation/docker-compose.md

```diff
@@ -69,13 +69,17 @@ docker-compose pull
 docker-compose up -d
 ```
 
-The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.
+The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file. We recommend that you edit the `.env` file and use the `AUTHENTIK_TAG` environment variable to override the default version defined in the compose file.
```

Comment:
It's actually _not_ recommended to override the version, as the recommended upgrade process downloads the latest docker-compose file which has the correct version defined

Reply:
I think I need to better understand that exact syntax that we want people to put into the .env file, then. I suspect I am being silly, since the whole purpose of a variable is that it is, well, variable, so in the .env we wouldn't want them to specify an exact version...? Ping me when you get a moment and let's discuss, pls.